src/container/srv_target.c

```diff
@@ -1373,11 +1373,7 @@ ds_cont_child_lookup(uuid_t pool_uuid, uuid_t cont_uuid,
 	if (rc != 0)
 		return rc;
 
-	if ((*ds_cont)->sc_stopping) {
-		cont_child_put(tls->dt_cont_cache, *ds_cont);
-		*ds_cont = NULL;
-		return -DER_SHUTDOWN;
-	}
+	D_ASSERT((*ds_cont)->sc_stopping == 0);
```

Comment:
im not sure this is safe actually. it failed in CI test:
#4  0x00007f3356f767bd in ds_cont_child_lookup (pool_uuid=0x7f3331a774b0 "7\235\205\r7ZG\006\214\262\353n Ð·\323.", cont_uuid=0x7f3331a77388 "\314$\200x\261\217N\213\255\037\242H\343D\343@", ds_cont=0x7f3331a771a0) at src/container/srv_target.c:1376

Reply:
Yup, while sc_stopping = 1, eg scrub ult might still try to lookup container which should fail. LRU evicting has not been called at this time.